### ISSUE TITLE
integrate `bindata_assetfs.go` into `go build ./cmd/leaps`

### ISSUE THREAD

**Issue by qknight** [Jeffail/leaps#33]
# problem desciption

at the moment we have duplicated work with the `www/ documents` because when using `-use_www ./cmd/leaps/www` we need them as normal documents in a filesystem but these exact files are in the git-repo twice actually.
## bindata_assetfs.go

`bindata_assetfs.go` is generated using these commands:

```
go-bindata-assetfs www/
gofmt -s -w bindata_assetfs.go
```

the resulting files contains a serialization of the `www/` directory. so any change in the `www/` directory has to be added manually to `bindata_assetfs.go` as well!
# proposed solution

generate `bindata_assetfs.go` on the fly and only update it when there were changes in the `www/` directory. this is pretty much as a `GNU Makefile` would work. i'm not sure how to do this with `go` but we have to find out.

**Comment by qknight** [comment]
@Jeffail what do you think about that? why is it the way it is currently, you surely had reasons to do so.

**Comment by Jeffail** [comment]
Hey @qknight, so my main concern was that the project should be buildable with only the basic Go toolset. Someone should be able to use `go get github.com/jeffail/leaps/cmd/leaps` with a clean installation of Go and the project should build and run just fine.

I actually once had a Makefile for this project, but I found that it became the core focus, and remaining compatible with `go get` seemed to be an afterthought which occasionally became broken. I chose to remove it and use `go generate` as it is a base feature of the Go toolset and therefore any build system a user has for Go projects should work in this case.

It would be nice if running `go build` would trigger the generate call automatically, but there doesn't seem to be way of doing that. It would also break the build for people who don't know to install `go-bindata`, and unless they are editing the `www` directory it shouldn't be a requirement (a lot of people will be building leaps simply to use it).
